setup.py

```diff
@@ -26,6 +28,6 @@
       license='BSD',
       packages=find_packages(exclude=['ez_setup', 'examples', 'tests']),
       include_package_data=True,
-      zip_safe=False,
-      extras_require = {'test':  ['pytest']}
+      zip_safe=True,
```

Comment:
This package should be zip safe, but for some reason a `.tar.gz` distilled is still built.  Maybe this is a Python or `setuptools` bug?  I'm running: Python 3.6.3 and setuptools 38.2.4.  Regardless, it seems are to leave this enabled.